webapp/components/Modal/ReportModal.vue

```diff
@@ -50,7 +50,6 @@
 <script>
 import { SweetalertIcon } from 'vue-sweetalert-icons'
 import { reportMutation } from '~/graphql/Moderation.js'
-import { reportReasonCategoriesDatabaseList } from '~/../shared/moderation/report.js'
```

Comment:
I think this didn't work in `jest` because of the how you are importing it @Tirokk 
why is there both a `~` and a `..`?? cause it lives in the root path? 
did you try `../../../shared/moderation/report.js`??

Reply:
Yes, for a pity I already have found that out and removed it.
We need a solution for that, in my eyes !!! @mattwr18